src/rank_llm/rerank/rank_gpt.py

```diff
@@ -144,20 +148,35 @@ def run_llm(
     def _get_prefix_for_rank_gpt_prompt(
```

Comment:
instead of introducing if/else in these functions please create separate helper functions:
e.g, _get_prefix_for_rank_gpt_apeer 

create_rank_gpt_prompt can call the proper functions based on prompt mode, this way your if/elses would be one-liners

Reply:
Done!